src/main/java/edu/illinois/library/cantaloupe/processor/AutomaticSelectionStrategy.java

```diff
@@ -11,8 +11,8 @@
 class AutomaticSelectionStrategy implements SelectionStrategy {
 
     private static final List<Class<? extends Processor>> JP2_CANDIDATES = List.of(
-            KakaduNativeProcessor.class,
```

Comment:
This here does not look right. KakaduNativeProcessor.class should not be removed, but instead OpenJpegProcessor.class if Java2d is the one handling it now

Reply:
Oh, good catch, thanks.